client/src/main/java/io/cloudsoft/winrm4j/client/WinRmClient.java

```diff
@@ -215,20 +285,99 @@ private WinRm getService() {
         }
     }
 
+    private class JaxWsClientWithNegotiateFactoryBean extends JaxWsClientFactoryBean {
+        @Override
+        protected Client createClient(Endpoint ep) {
+            return new ClientImpl(getBus(), ep, getConduitSelector());
+        }
+
+    }
+
     private synchronized WinRm createService() {
         if (winrm != null) return winrm;
 
         WinRmService service = new WinRmService();
+//        JaxWsDynamicClientFactory dcf = JaxWsDynamicClientFactory.newInstance();
+//        Client client = dcf.createClient("people.wsdl", classLoader);
         winrm = service.getWinRmPort(
                 // * Adds WS-Addressing headers and uses the submission spec namespace
                 //   http://schemas.xmlsoap.org/ws/2004/08/addressing
                 newMemberSubmissionAddressingFeature());
 
+        // Needed to be async according to http://cxf.apache.org/docs/asynchronous-client-http-transport.html
+//        Bus bus = BusFactory.getDefaultBus();
+//        bus.setProperty(AsyncHTTPConduit.USE_ASYNC, Boolean.TRUE);
+
+        Client client = ClientProxy.getClient(winrm);
+        ServiceInfo si = client.getEndpoint().getEndpointInfo().getService();
+        si.setProperty("soap.force.doclit.bare", true);
+        si.setProperty("soap.no.validate.parts", true);
```

Comment:
Are these properties still needed after latest changes? If so could you add short comments for the reason why they are needed.